viewer/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1042,6 +1045,82 @@ export class Cognite3DViewer {
     throw new NotSupportedInMigrationWrapperError('Cache is not supported');
   }
 
+  /**
+   * Creates a tool for attaching HTML elements to a 3D position and updates it's
+   * position/visibility as user moves the camera. This is useful to create HTML
+   * overlays to highlight information about key positions in the 3D model.
+   *
+   * Attached elements *must* have CSS style 'position: absolute'. It's also recommended
+   * in most cases to have styles 'pointerEvents: none' and 'touchAction: none' to avoid
+   * interfering with 3D navigation. Consider also applying 'transform: translate(-50%, -50%)'
+   * to anchor the center of the element rather than the top-left corner. In some cases the
+   * `zIndex`-attribute is necessary for the element to appear on top of the viewer.
```

Comment:
you described z-index but don't set it below. I would actually suggest having as little CSS as possible in examples. Especially if you provide tips in plain text. Or vice versa, move the plain text to be comments for code. That will give you less inconsistency between duplicated information. I would say it's better to use code comments (easier to copy-paste the same to the docs) and doesn't require additional explanation.